agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -93,14 +93,26 @@ while true; do
 		;;
 	esac
 done
+
+function setup_ara {
+	generate_inventory >> /tmp/inventory.$$
+	check_install_rpm pbench-ara-role
+	ansible-playbook -i ${INVENTORY} /opt/pbench-agent/ansible/ara/pbench-ara.yml &>$pbench_log
+}
 	
 if $check; then
-	if [ "$sysinfo" == "all" ] || [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
+	if [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
 		:
+	elif [ "$sysinfo" == "all" ]; then
+		# generate inventory with pbench controller and remotes, install and configure ara
+		setup_ara
```

Comment:
Can we see what needs to be setup for ara first?  Is that posted in this commit somewhere?

Second, we need do need some kind of collection setup, why not make a generic "`setup_<tool>`" and invoke it just like we do the other sysinfo options?